idle_sweep_test.go

```diff
@@ -362,7 +364,14 @@ func TestIdleSweepIgnoresConnectionsWithCalls(t *testing.T) {
 		// Unblock the call.
 		close(block)
 		<-c2CallComplete
-		check.tick()
+
```

Comment:
similar to `waitForZeroConnections`, an alternative would be to add a `waitForZeroExchanges` that uses introspection to check the number of pending message exchanges and only does `check.tick()` once the exchanges are 0?

that way it should avoid the race completely (vs mitigating the race by running more ticks)

Reply:
added as suggested. thanks!